### PR TITLE
Update runtime

### PR DESCRIPTION
**Pull request by JDragovichAlertLogic**

### CODE REVIEW

File: checkin.js [hunk 1]
```diff
@@ -11,34 +11,12 @@
 const AWS = require('aws-sdk');
 const async = require('async');
 const m_alServiceC = require('al-collector-js/al_servicec');
+const m_alAzCollectC = require('al-collector-js/azcollectc');
 const m_packageJson = require('./package.json');
 
 const AZCOLLECT_ENDPOINT = process.env.azollect_api;
 const COLLECTOR_TYPE = 'cwe';
 
-function checkCloudFormationStatus(event, callback) {
```
Comment (kkuzmin) [inline checkin.js]: Is this check moved into the `al-aws-collector-js`?

File: checkin.js [hunk 2]
```diff
@@ -11,34 +11,12 @@
 const AWS = require('aws-sdk');
 const async = require('async');
 const m_alServiceC = require('al-collector-js/al_servicec');
+const m_alAzCollectC = require('al-collector-js/azcollectc');
```
Comment (kkuzmin) [inline checkin.js]: Looks like the two vars above aren't used anywhere.

File: index.js [hunk 3]
```diff
@@ -17,7 +17,10 @@ const async = require('async');
 const zlib = require('zlib');
 
 const m_alServiceC = require('al-collector-js/al_servicec');
+const m_alAzCollectC = require('al-collector-js/azcollectc');
```
Comment (kkuzmin) [inline index.js]: the two vars above aren't used anywhere.

File: checkin.js (hunk 2 above)
Comment (JDragovichAlertLogic) [reply checkin.js]: not sure why the linter didnt catch that., but youre right.

File: checkin.js (hunk 1 above)
Comment (JDragovichAlertLogic) [reply checkin.js]: Yep, its the only default heatlh check in al-aws-collector-js

https://github.com/alertlogic/al-aws-collector-js/blob/master/health_checks.js#L31-L51

File: index.js (hunk 3 above)
Comment (JDragovichAlertLogic) [reply index.js]: Those and more apprently. deleting those suddenly made the linter wake up and show a few more...

**Pull request merged**